### PR TITLE
DCM KIT Nesting Content

### PR DESCRIPTION
**Pull request by Frank-Globisch**
Added nesting as use case in DCM Kit and included updated references and changed break codes

### CODE REVIEW

**Review by lanfrans — APPROVED** [review]

File: docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+
+Consolidation is performed only on the demand and not on the capacity side when Nesting is used: all related capacity information is fully independent, i.e. the capacity data in the Capacity Group must be maintained individually. If a Capacity Group contains linked Capacity Groups, then the Material Demands of these linked Capacity Groups are summed up and used as aggregated demand. The supplier must either link individual Material Demands or link existing Capacity Groups in the Capacity Group. <br/>
+  
+
+The following figure shows that:  
+- Capacity Group 1 and 2 are linked to Capacity Group 3 (i.e. Nesting or indirect linking).  
+- Material Demand 1 and 2 are linked to Capacity Group 1 (i.e. direct linking). 
+- Material Demand 3 and 4 are linked to Capacity Group 2 (i.e. direct linking). 
+- Capacity Group 3 considers the Material Demands 1, 2, 3 and 4 because they are indirectly linked via the nested Capacity Group 
+
```
Comment (ClosedSourcerer) [inline docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md:59]: Please add SVGs for the BLockBeta Code.
Also remeber to add license Files for every picture.

File: docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+```
+-->
+Figure: *Capacity Group with directly linked Material Demands or nested Capacity Groups (indirectly linked Material Demands)*  
+
+
+Capacity Group 3 is the result of the indirect way of linking Material Demand, therefore it must not contain any additional directly linked Material Demands.  
+
+If a linked Capacity Group contains further linked Capacity Groups (dark green in figure below), the Nesting is recurrent and must go further until referenced Capacity Groups contain only linked demand series (orange in figure below) ("domino effect").
+
+![Recurrent nested Capacity Groups" src="!](./resources/2024-09-24_DCM_Nesting_example_picture.svg)
```
Comment (ClosedSourcerer) [inline docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md:118]: License File missing for this picture.
Please Keep the naming scheme consistent with the existing ressources

File: docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+- Allows to provide additional transparency to customers
+
+<br/><br/>
+
+## Functional description
+
+Nesting is an indirect linking of Material Demand information to a Capacity Group done by a supplier. It involves connecting the WeekBasedCapacityGroup to another WeekBasedCapacityGroup that is already linked to a WeekBasedMaterialDemand.
+Suppliers may apply Nesting within WeekBasedCapacityGroup aspect model via the parameter 
+
+```
```
Comment (ClosedSourcerer) [inline docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md:41]: I do not think that a whole codeblock ``` should be used instead maybe use a single backtick ` to put emphasize on linkedCapacityGroups.

At the same time both the CodeBlock as well as Background make the "" redundant
```
linkedCapacityGroups
```

`linkedCapacityGroups`

File: docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md [hunk 4]
```diff
@@ -0,0 +1,276 @@
...
+The same applies to multiple production plants representing a customer's total allocated capacity for materials.
+<br/><br/>
+The supplier is able to put the desired Capacity Groups with their respective demands and capacities together and then share it with the customer.<br/>
+
+Consolidation is performed only on the demand and not on the capacity side when Nesting is used: all related capacity information is fully independent, i.e. the capacity data in the Capacity Group must be maintained individually. If a Capacity Group contains linked Capacity Groups, then the Material Demands of these linked Capacity Groups are summed up and used as aggregated demand. The supplier must either link individual Material Demands or link existing Capacity Groups in the Capacity Group. <br/>
+  
+
+The following figure shows that:  
+- Capacity Group 1 and 2 are linked to Capacity Group 3 (i.e. Nesting or indirect linking).
```
Comment (ClosedSourcerer) [inline docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md:55]: Consistency: 
In the Figure there is no `Capacity Group 1` just a `Capacity Group - 1`

File: docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md [hunk 5]
```diff
@@ -0,0 +1,276 @@
...
+If a linked Capacity Group contains further linked Capacity Groups (dark green in figure below), the Nesting is recurrent and must go further until referenced Capacity Groups contain only linked demand series (orange in figure below) ("domino effect").
+
+![Recurrent nested Capacity Groups" src="!](./resources/2024-09-24_DCM_Nesting_example_picture.svg)
+Figure: *Recurrent nested Capacity Groups*
+<br/><br/>
+
+**Considerations**
+
+- It must not be possible to have a combination of directly linked Material Demand series and linked Capacity Groups (Nesting) within the same Capacity Group.
+- Suppliers can use comments to provide customers with additional information about the Nesting. For more details on this communication feature, see Chapter 5.9 in the [CX-0128 DCM Standard document](https://catenax-ev.github.io/docs/standards/CX-0128-DemandandCapacityManagementDataExchange) or [Comments](https://eclipse-tractusx.github.io/docs-kits/kits/DCM-Kit/adoption-view/use-case/comments) chapter in DCM Kit of Tractus-X.
```
Comment (ClosedSourcerer) [inline docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md:125]: chapter not Chapter.
For reference: https://github.com/catenax-eV/cx-ex-dcm/tree/main/CX-0128 x.y is a chapter x.y.z would be a section.

Comment (ClosedSourcerer) [inline docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md:125]: `chapter in DCM Kit of Tractus-X.` 
The KIT is not structured into chapters and sections, but rather pages.

File: docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md [hunk 6]
```diff
@@ -0,0 +1,276 @@
...
+- Suppliers should be carefully applying recurring linked Capacity Groups ("domino effect"), because of the complexity of keeping track on changes and in maintenance. It is more difficult to identify and understand demand changes and where they are originating from (e.g. which part).
+
+<br/><br/>
+
+Feature Nesting is to be used optional but if it is used certain parameters are required to be filled.
+
+__Parameters of the Nesting are as follows:__
+
+| Main Parameters | Required? | Description | Example |
```
Comment (ClosedSourcerer) [inline docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md:134]: Just as a comment:
From a technical perspective there is mandatory and optional.

File: docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md (hunk 3 above)
Comment (LukasSchmetz) [reply docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md:41]: Remark: We decided to fully removed the codeblock in chapter "Load Factors" after Philipps identical comment. Most important thing to stay consistent for all features.

File: docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md [hunk 7]
```diff
@@ -0,0 +1,276 @@
...
+title: Nesting
+description: Nest material demands and capacity groups into other capacity groups
+sidebar_position: 8
+---
+
+![DCM kit banner](/img/kit-icons/dcm-kit-icon.svg)
+
+## Business Roles and Functions
+
+Nesting is a feature defined in the DCM standard CX-0128 and includes an advanced property of the Capacity Group structure and therefore may be used optionally by suppliers. The application of Nesting is embedded into the WeekBasedCapacityGroup aspect model. This means that only suppliers may apply Nesting and customers consume it.
```
Comment (LukasSchmetz) [inline docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md:12]: Add direct Link to CX-0128?

Comment (LukasSchmetz) [reply docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md:12]: "The application of Nesting" => Vorschlag "The Nesting feature"

File: docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md [hunk 8]
```diff
@@ -0,0 +1,276 @@
...
+
+|Function / Role|Customer|Supplier|
+|-|-|-|
+|Define Nesting in a Capacity Group||X|
+|Inform business partners about Nesting via data transfer||X|
+|Receive Nesting information|X||
+
+<br/><br/>
+
+## Business value
```
Comment (LukasSchmetz) [inline docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md:22]: Consistency in headlines (e.g. "Business Roles and Functions" vs. "Business value" => Proposal to use capital letters "Business Value","Functional Description" etc

File: docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md (hunk 3 above)
Comment (LukasSchmetz) [reply docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md:41]: Proposal to integrate it as a parameter table like in the other features and aligned in workshop

File: docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md (hunk 1 above)
Comment (LukasSchmetz) [reply docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md:59]: i.e. can be removed within the brackets from my point of view. "." to be used or removed for all bullet points to keep it consistent.

File: docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md [hunk 9]
```diff
@@ -0,0 +1,276 @@
...
+```mermaid
+block-beta
+columns 4
+A["Demand data (directly linked)"] style A fill:#FFA600,color:#000000
+B["Demand data (indirectly linked)"] style B fill:#BF7100,color:#F4F2F3,stroke-dasharray:3
+C["Capacity data"] style C fill:#B3CB2D,color:#000000
+D["Capacity data (linked or nested)"] style D fill:#617000,color:#F4F2F3,stroke-dasharray:3
+```
+-->
+Figure: *Example for application of Nesting*
```
Comment (LukasSchmetz) [inline docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md:253]: svg to be integrated as currently no figure to be seen

File: docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md (hunk 2 above)
Comment (MWADCM) [reply docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md:118]: @ClosedSourcerer  License File issue solved?

File: docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md (hunk 4 above)
Comment (MWADCM) [reply docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md:55]: Classical Mistake. Need to be corrected.

**Review by KZI-CatX — APPROVED** [review]

File: docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md [hunk 10]
```diff
@@ -0,0 +1,274 @@
...
+
+## Business value
+
+Nesting may be used by suppliers for the following purposes (but not limited to):
+
+- To simplify the maintenance of Capacity Groups
+- To enable the bundling of several smaller Capacity Groups to reflect a (virtual, internal) company structure
+
+<br/><br/>
+
```
Comment (ClosedSourcerer) [inline docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md:30]: Maybe remove the breaks?
Do we really need them outside of tables?

File: docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md [hunk 11]
```diff
@@ -0,0 +1,274 @@
...
+**Advantages**
+
+- Avoid increased maintenance efforts if same Material Demands may be required for another Capacity Group
+- Allows to provide additional transparency to customers
+
+<br/><br/>
```
Comment (ClosedSourcerer) [inline docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md:36]: Another break

**Review by ClosedSourcerer — CHANGES_REQUESTED** [review]
Found additional stuff, please adress.

File: docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md (hunk 10 above)
Comment (ClosedSourcerer) [reply docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md:30]: Fixed it via commit: https://github.com/ClosedSourcerer/eclipse-tractusx.github.io/pull/18/commits/c7d7188e09d84bea76888b2f0ab5beb1461f10c7

**Review by sarahmater — APPROVED** [review]

**Review by MWADCM — APPROVED** [review]

File: docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md (hunk 7 above)
Comment (Frank-Globisch) [reply docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md]: Updated wording as suggested.

File: docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md (hunk 8 above)
Comment (Frank-Globisch) [reply docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md]: Updated in PR [https://github.com/ClosedSourcerer/eclipse-tractusx.github.io/commit/a99a7ba9bef353b87f35013562f173b7ec0e5017](https://github.com/ClosedSourcerer/eclipse-tractusx.github.io/commit/a99a7ba9bef353b87f35013562f173b7ec0e5017)

File: docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md (hunk 3 above)
Comment (Frank-Globisch) [reply docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md]: The parameter table is already available further below in the description. Therefore I only changed the codeblock to a single backtick as suggested. PR [https://github.com/ClosedSourcerer/eclipse-tractusx.github.io/commit/2342a507fc08bbd637203a30b0dab373eaf4867e](https://github.com/ClosedSourcerer/eclipse-tractusx.github.io/commit/2342a507fc08bbd637203a30b0dab373eaf4867e)

File: docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md (hunk 1 above)
Comment (Frank-Globisch) [reply docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md:57]: SVGs were inserted via PR [https://github.com/ClosedSourcerer/eclipse-tractusx.github.io/commit/b1d90cef92d3e13caa9c9a20b83488417ee951ed](https://github.com/ClosedSourcerer/eclipse-tractusx.github.io/commit/b1d90cef92d3e13caa9c9a20b83488417ee951ed). I noticed that the coloring doesn't fit completely, thus will not mark it as resolved yet.

**Review by Frank-Globisch — COMMENTED** [review]
Updated nesting.md based on several review comments (not completely yet).

**Comment by ClosedSourcerer** [comment]
> Updated nesting.md based on several review comments (not completely yet).

The inserted SVGs are no vectorgraphics, but rather base 64 jpegs embedded into an SVG file --> They do not scale.
Please add true scalable vector graphics.

File: docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md (hunk 1 above)
Comment (Frank-Globisch) [reply docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md:57]: SVGs with correct transparency and colors as well as fitting font size were uploaded together with license files.

File: docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md (hunk 2 above)
Comment (Frank-Globisch) [reply docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md:118]: Updated picture with naming scheme was uploaded a few days back 
License file was added today [https://github.com/ClosedSourcerer/eclipse-tractusx.github.io/pull/18/commits/fa0c140c9314cd03e0b1b40bee65123a076f4beb](https://github.com/ClosedSourcerer/eclipse-tractusx.github.io/pull/18/commits/fa0c140c9314cd03e0b1b40bee65123a076f4beb)

File: docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md (hunk 5 above)
Comment (Frank-Globisch) [reply docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md]: Updated in PR [https://github.com/ClosedSourcerer/eclipse-tractusx.github.io/commit/f5df057e5071fb84e7e8b2cb074e66e7b9dec852](https://github.com/ClosedSourcerer/eclipse-tractusx.github.io/commit/f5df057e5071fb84e7e8b2cb074e66e7b9dec852)

File: docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md (hunk 6 above)
Comment (Frank-Globisch) [reply docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md:134]: Understand, however checking in other branches I noticed that they also still now use "Yes" and "No", thus no change.

File: docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md (hunk 4 above)
Comment (Frank-Globisch) [reply docs-kits/kits/Demand and Capacity Management Kit/adoption-view/use-case/nesting.md:53]: Please check - IMO this is resolved with the past updates to the svg pictures. Latest svg update PR [https://github.com/ClosedSourcerer/eclipse-tractusx.github.io/pull/18/commits/7f673c81012479cf5d4f22f8c789c287eecee25f](https://github.com/ClosedSourcerer/eclipse-tractusx.github.io/pull/18/commits/7f673c81012479cf5d4f22f8c789c287eecee25f)

**Comment by AnastasiaNogay** [comment]
looks good.

**Review by ClosedSourcerer — CHANGES_REQUESTED** [review]
Please have a close look at every single license notice.

1. Which companies did actually contribute to nesting in an IP-worthy way?
2. I edited the notice within nesting.md, expressing my very generous perception.

The SVGs. do not use the Font Manrope.

1. Please provide me with the original file (for svgs that are not derived from mermaid code) and i will make the appropriate changes as well as save the source file to expert group github for further use

@Frank-Globisch

**Review by LukasSchmetz — APPROVED** [review]
LGTM

**Comment by Frank-Globisch** [comment]
> Please have a close look at every single license notice.
> 
> 1. Which companies did actually contribute to nesting in an IP-worthy way?
> 2. I edited the notice within nesting.md, expressing my very generous perception.
> 
> The SVGs. do not use the Font Manrope.
> 
> 1. Please provide me with the original file (for svgs that are not derived from mermaid code) and i will make the appropriate changes as well as save the source file to expert group github for further use
> 
> @Frank-Globisch

Hi @ClosedSourcerer ,
original file was shared with you via email a few minutes ago as requested.
As for the notice: I'm fine either way, however I had updated the notice just yesterday after alignment with @LukasSchmetz and thus I think it would be good to have a general rule how to specify the licence notice. Maybe this can be clarified in our weekly call tomorrow ?

**Review by AnastasiaNogay — APPROVED** [review]
reviewed, looks perfect.

**Pull request merged**